c2corg_ui/templates/xreport/helpers/view.html

```diff
@@ -0,0 +1,153 @@
+<%def name="show_fulldate(date)">\
+  % if date:
+    <span class="date-locale">{{ '${date}' | amDateFormat:"dddd Do MMMM YYYY" }}</span>
+  % endif
+</%def>
+
+<%def name="get_xreport_general(xreport)">\
+<div class="name-icon-value general" ng-click="detailsCtrl.openTab($event)">
+  <h4>
+    <span translate>General</span><span class="glyphicon glyphicon-menu-right"></span>
+  </h4>
+  <div class="name-icon">
+    <span class="glyphicon glyphicon-info-sign"></span>
+    <p translate>General</p>
+  </div>
+  <span class="detail-text accordion">
+
+    % if xreport.get('activities'):
+      <article class="value activities">
+          <span translate class="value-title">activities</span><br>
+        % for activity in xreport.get('activities') :
+          <span class="route-activity icon-${activity}" uib-tooltip="{{mainCtrl.translate('${activity}')}}"></span>
+        % endfor
+      </article>
+    % endif
+
+    % if xreport.get('event_type'):
+      <article class="value activities">
+          <span translate class="value-title">event_type</span><br>
```

Comment:
I see  `.value, .value-title {  font-weight: bold; }`  still in Less file... I will add `:`